server/sendemail.ts

```diff
@@ -0,0 +1,51 @@
+import { SES } from "aws-sdk";
+import { SendEmailRequest } from "aws-sdk/clients/ses";
+
+export const sendEmail = (email: string) => {
+  try {
+    const REGION = "us-east-1";
+
+    const charset = "UTF-8";
+
+    const html = `<html>
+      <head>
+      <title>Page Title</title>
+      <style>
+      </style>
+      </head>
+      <body>
+      <div style=font-family: Arial, Helvetica, sans-serif;">
+      <img src="kcsa_logo.png" alt="KCSA Logo" style="width:200px">
+      <h1>Welcome to KCSA!</h1>
+      <p>You have created an account with us at KCSA. To verify your email, please click the link below.</p>
+      <a href="http://kcsacanada.ca/">http://kcsacanada.ca/</a>
+      
+      </div>
+      </body>
+      </html>`;
```

Comment:
this is okay for now but could you look into how we can style out emails?

Reply:
okok